src/git_push.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
```

Comment:
Why do we need this script?

Reply:
We don't need this, semantic-release couldn't run generate.sh file